### ISSUE TITLE
bundling failed: "Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by paulo2nd** [futurepress/react-native-static-server#17]
Every time I add react-native-static-server into my project I got the error: bundling failed: "Unable to resolve module `AccessibilityInfo`.

I'm  using react-native@0.46.4, tried everything, nothing works...

**Comment by paulo2nd** [comment]
started a new project, forked this repo and removed old react-native dependency... everything working right now.
